docs/Get started/why-fleet.md

```diff
@@ -4,9 +4,9 @@ Fleet is an open-source device management platform for Linux, macOS, Windows, Ch
 
 ## What's it for?
 
-Managing computers today is getting harder. You have to juggle a mix of operating systems and devices, with a whole bunch of middleman vendors in between. 
+Managing computers today is getting harder. You have to juggle a mix of operating systems and devices, with a bunch of middleman vendors in between. 
```

Comment:
Maybe it’s better to keep “whole”, reads easier even though slightly longer

Reply:
I think it's better this way